Bottom: fa9e2a2b8f0c41317112f887c95fd1de4667e40a
Top:    e0bd969bb85c70057e908c26f04cb5a70425072b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 16:04:49 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 6f54d82..ace594a 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -567,8 +567,6 @@ ghid_screen_update (void)
 #endif
 }
 
-void DrawAttached (bool);
-
 #define Z_NEAR 3.0
 gboolean
 ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
